foundation-extensions/mobile-core/configuration/README.md

```diff
@@ -1,18 +1,18 @@
 # Configuration
 
-The Configuration extension is built into the Mobile Core extension. It provides several different APIs for you to setup the configuration either remotely on Experience Platform Launch or locally.
+The Configuration extension is built into the Mobile Core extension. It provides several different APIs for you to setup the configuration either remotely in the Data Collection UI or locally.
 
-## Configure with the Experience Platform Launch App ID
+## Configure with the Experience Platform App ID
```

Comment:
Experience Platform App ID/environment is little confusing as these terms does not show anywhere in the UI.
@calebk1 I propose we change these occurrences to something like this:
 - `Configure with the Experience Platform App ID` -> `Configure with App ID per environment`
- `Adobe Experience Platform generates a unique environment ID` -> `When you configure a mobile property, a unique environment ID is generated that the SDK uses to retrieve ...`
- `We strongly recommend that you configure the SDK with the Experience Platform environment ID` -> We strongly recommend that you configure a mobile property in the Data Collection UI and use environment IDs to configure your application.`
What are your thoughts on this?

Reply:
I know according to the Adobe style guidelines, we're supposed to avoid usage of "We" in the docs, which is why I wrote out the we part.

How about "As best practice, you should configure a mobile property in the Data Collection UI and use environment IDs to configure your application."?

Reply:
Thanks for the inputs @emdobrin , @calebk1 . I will incorporate them in the upcoming revision.

Reply:
@prudrabhat it would be great if you can update these in current PR at least for the modified files and I am ok if you want to create a new PR for updating other files if there are too many changes.